envoy/network/listener.h

```diff
@@ -429,6 +434,11 @@ class Listener {
    */
   virtual void setRejectFraction(UnitFloat reject_fraction) PURE;
 
+  /**
+   * Check whether the listener should bypass overload manager actions
+   */
+  virtual bool getBypassOverloadManager() PURE;
```

Comment:
this is kind of confusingly named since we're not returning a overload manager or so. e.g. getX doesn't return X.

Consider: `shouldBypassOverloadManager` or similar if we need to return a bool